policyengine_us/parameters/gov/states/ga/tax/income/agi/subtractions.yaml

```
@@ -0,0 +1,45 @@
+description: Georgia subtracts the following from the adjusted gross income.
+values:
+  2021-01-01:
+    - us_govt_interest
+    # ga_retirement_income_exclusion
+    # Military retirement income subtraction
+    # Dependent personal exemption for unborn children with a detectable human heartbeat
+    # Social security or railroad retirement
+    # Salaries and wages
+    # Individual retirement account, Keogh, SEP and SUB-S plan withdrawals
+    # Depreciation because of differences
+    # Dependent’s unearned income
```

i think the unicode windows error results from these apostrophes. try this here and elsewhere
```suggestion
    # Dependent's unearned income
```